sdks/java/io/hadoop/input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIOTest.java

```diff
@@ -472,6 +472,7 @@ public void testReadWithNullCreateRecordReader() throws Exception {
    */
```

Comment:
it seems like we're changing this file in both PRs - we should pick one, otherwise there'll be a merge conflict